release_bot/releasebot.py

```diff
@@ -54,6 +54,7 @@ def cleanup(self):
         self.new_pr = {}
         self.github.comment = []
         self.fedora.progress_log = []
+        self.git.cleanup()
```

Comment:
The commit comment says "_don't erase the cloned repo after creating a PR_", but it looks to me that `ReleaseBot.cleanup()` is called after `ReleaseBot.make_release_pull_request()`.

Reply:
The problem was that ReleaseBot.git was cleaned up after the PR was created - this broke tests because of the way they written. Hence I moved the cleanup logic to the infinite loop so that the git repo can be utilized.